WordPress/gutenberg
# Quote Block: Add align support

Part of #43248
Related to #11610

## What?

This PR adds the `align` support to the Quote block.

## Why?

In #11610, it is suggested that the Pullquote block may be deprecated. If the Pullquote block is deprecated, the Quote block must support as many of the same features as possible. The most important feature that the Quote block is missing is `align` support.

## How?

Simply added algin support.

Currently, this block does not support layout, so the content width will be the full width of the block. As a follow-up, I'm considering adding layout support to this block as well, as in #45326.

## Testing Instructions

- Try applying different align layouts to this block.
- When converting the wide/full-width Pull quote block to the Quote block, the width should carry over.

## Screenshots or screencast 

![quote](https://github.com/user-attachments/assets/2a352b68-1583-4a1b-aec8-0caf1df932f3)
